src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -152,7 +156,15 @@ const NotificationSettings = ({
                 </NotificationGroup>
               );
             }
-
+            // this will hide the Payments header when there are no items to display
+            if (
+              id === NOTIFICATION_GROUPS.PAYMENTS &&
+              !toggles.profileShowNewBenefitOverpaymentDebtNotificationSetting &&
+              !toggles.profileShowNewHealthCareCopayBillNotificationSetting &&
+              !mobilePhoneNumber
+            ) {
+              return null;
+            }
```

Comment:
nice job figuring this out. This is another case where if we could do this instead on the BE api side then we wouldn't have to hop through all these filtering hoops... but this will work for now.